src/backend/task/pg_cron.c

```diff
@@ -910,8 +910,8 @@ PollForTasks(List *taskList)
 		}
 
 		pollFileDescriptor->revents = 0;
-
-		activeTaskCount++;
+		if (pollFileDescriptor->fd > 0)
```

Comment:
The document https://www.postgresql.org/docs/14/libpq-status.html says that a valid file desc is greater than or equal to 0.

Reply:
strictly, yes, done.